src/lib/i18n.js

```diff
@@ -26,3 +28,4 @@ const createAnnotatorIntl = () => {
 };
 
 export default { createAnnotatorIntl };
+export { currentLocale };
```

Comment:
Please add this to the object in the default export above to keep the module exports consistent.

Reply:
This is already in that object (under `locale` property). The problem is that I don't know how to access this object in my React components. I see it declared in `BaseViewer.js`, but don't see it being used in any of React components.

Reply:
To clarify, I'm just asking that we move the export to the default:

```
export default const { createAnnotatorIntl, currentLocale };`

Reply:
Then I misunderstood. I'll do that then.